### ISSUE TITLE
Return support to React 16^

### ISSUE THREAD

**Issue by BilyachenkoOY** [maslianok/react-resize-detector#132]
It would be great to update package.json to support react & react-dom of both version: `^16.0.0 || ^17.0.0` since there are no breaking changes affecting react-resize-detector in version 17.

The motivation for this is that many packages have not updated their dependencies to v17 (while there are already some packages which support both versions), so there will be peer dependencies conflicts.

This should work fine:
```
  "peerDependencies": {
    "react": "^16.0.0 || ^17.0.0",
    "react-dom": "^16.0.0 || ^17.0.0"
  }
```

**Comment by maslianok** [comment]
Definitely a good idea! Thanks 👍 

Published in v6.3.3

**Comment by BilyachenkoOY** [comment]
This is the fastest support I've ever seen. Thank you.
